server/game/cards/10-BMR/TheLawGoesUnderground.js

```diff
@@ -0,0 +1,45 @@
+const ActionCard = require('../../actioncard.js');
+const GameActions = require('../../GameActions');
+/** @typedef {import('../../AbilityDsl')} AbilityDsl */
+
+class TheLawGoesUnderground extends ActionCard {
+    /** @param {AbilityDsl} ability */
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'The Law Goes Underground',
+            playType: ['resolution'],
+            cost: ability.costs.choose({
+                'Boot Stud': ability.costs.boot({
+                    type: 'dude',
+                    condition: card => this.game.shootout && card.isStud() && this.game.shootout.getPosseByPlayer(this.controller).isInPosse(card)
+                }),
+                'Discard Stud': ability.costs.discardFromPlay({
+                    type: 'dude',
+                    condition: card => this.game.shootout && card.isStud() && this.game.shootout.getPosseByPlayer(this.controller).isInPosse(card)
+                })
+            }),
+            message: context => this.game.addMessage('{0} uses {1} to flee to their hideout', context.player, this),
+            handler: context => {
+                const actor = context.costs.boot ? context.costs.boot : context.costs.discardFromPlay;
+                const actorIsDeputy = actor.hasKeyword('deputy');
+                if(actorIsDeputy) {
+                    const opposingShooter = this.game.shootout.getPosseByPlayer(context.player.getOpponent()).shooter;
+                    this.game.resolveGameAction(GameActions.sendHome({ card: opposingShooter, options: { needToBoot: true } }), context);
+                    this.game.addMessage('{0} sends {1} home because {2} is a Deputy', context.player, opposingShooter, actor);
```

Comment:
Use a card in the message so if someone will read the chat log later it is clear what cards caused it.
`Player uses X to do Y`

Reply:
done